app/bundles/CoreBundle/Config/config.php

```diff
@@ -1835,6 +1835,10 @@
         'transliterate_page_title'  => false,
         'cors_restrict_domains'     => true,
         'cors_valid_domains'        => [],
+        'headers_sts'               => false,
+        'headers_sts_expire_time'   => 31536000,
```

Comment:
wouldn't it make sense to set the default the same value in every file?
Here it it 31536000 (1 year), all other places 60 (1 minute).

I suggest to change the default to 60 to keep it consistent (and safe as default value)

```suggestion
        'headers_sts_expire_time'   => 60,
```